Platform/ElkhartlakeBoardPkg/CfgData/CfgData_BootOption.yaml

```diff
@@ -14,11 +14,16 @@
 - !expand { BOOT_OPTION_TMPL : [ 2  , 0x1F   ,  16 ,    5   ,   0   ,    0  ,     1 ,     1 , '/boot/sbl_rtcm' ] }
 - !expand { BOOT_OPTION_TMPL : [ 3  ,   0    ,  20 ,    0   ,   0   ,    1  ,     1 ,     1 , '/boot/sbl_os' ] }
 - !expand { BOOT_OPTION_TMPL : [ 4  , 0x9E   ,  16 ,    0   ,   0   ,    1  ,     1 ,     1 , '!IPFW/POSC' ] }
-- !expand { BOOT_OPTION_TMPL : [ 5  ,   0    ,  20 ,    2   ,   0   ,    0  ,     1 ,     1 , '/boot/sbl_os' ] }
-- !expand { BOOT_OPTION_TMPL : [ 6  , 0x9E   ,  16 ,    0   ,   0   ,    1  ,     1 ,     1 , '!IPFW/POSC' ] }
-- !expand { BOOT_OPTION_TMPL : [ 7  ,   0    ,  20 ,    6   ,   0   ,    0  ,     1 ,     1 , '/boot/sbl_os' ] }
-- !expand { BOOT_OPTION_TMPL : [ 8  , 0x9E   ,  16 ,    0   ,   0   ,    1  ,     1 ,     1 , '!IPFW/POSC' ] }
-- !expand { BOOT_OPTION_TMPL : [ 9  ,   0    ,  20 ,    3   ,   0   ,    0  ,     1 ,     1 , '/boot/sbl_os' ] }
+- !expand { BOOT_OPTION_TMPL : [ 5  , 0x1F   ,  16 ,    0   ,   0   ,    0  ,     1 ,     1 , '/boot/sbl_rtcm' ] }

```

Comment:
Normally, 1st boot option is default boot media which has high data transfer speed and it's only available on release build. Why USB is the default boot media instead of NVMe or UFS?

Reply:
Agree and this make more sense for default booting with faster device